### PR TITLE
AJ Sappenfield's Contribution

### PR DESCRIPTION
**Pull request by sappenfield**
My first contribution and pull request!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -122,6 +122,40 @@ <h2>A project for first-time contributions</h2>
...
+        <!-- ________ Sappenfield's Contributor card START ________  -->
+        <div class="card">
+          <p class="name">Albert J. Sappendield</p>
+          <p class="contact">
+            <i class="fab fa-x-twitter"></i>
+            <a href="https://twitter.com/aj_sappenfield" target="_blank">aj_sappenfield</a>
+            <i class="fab fa-linkedin"></i>
+            <a href="http://www.linkedin.com/in/a-j-sappenfield/" target="_blank">AJ Sappenfield</a>
+            <i class="fab fa-github"></i>
+            <a href="https://www.github.com" target="_blank">sappenfield</a>
```
Comment (Amazening) [inline index.html:134]: Thank you for your contribution! Before I merge your pull request, there's just one little thing to fix. The GitHub link you gave is to the main GitHub page, as opposed to your specific GitHub account. Once you fix that, I'll merge your request.

File: index.html [hunk 2]
```diff
@@ -122,6 +122,40 @@ <h2>A project for first-time contributions</h2>
...
+            expertise and project management skills to contribute to groundbreaking advancements in aerospace technologies. 
+            </p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="https://stackoverflow.com/" target="_blank" title="First Resource">Stack Overflow ftw</a>
+              </li>
+              <li>
+                <a href="https://leetcode.com/problems/two-sum/" target="_blank" title="Second Resource">Leetcode a day keeps the depression away</a>
```
Comment (Amazening) [inline index.html:149]: Not sure if this is intentional or not (it should be fine either way) but the link here goes directly to the two sum problem, as opposed to the landing page for leetcode. Feel free to fix that if it's a mistake, or leave it if it isn't.

**Review by Amazening — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (sappenfield) [reply index.html:149]: It was intentional, I find that specific problem is an important intro into algorithms lol

File: index.html (hunk 1 above)
Comment (sappenfield) [reply index.html]: Has been fixed! Thanks for pointing that out!

File: index.html (hunk 2 above)
Comment (Amazening) [reply index.html:149]: Awesome, figured I'd check.

**Comment by Amazening** [comment]
Thank you for your pull request. It has now been merged and should be live in a few minutes. Congratulations, and feel free to check out our [next steps](https://github.com/Syknapse/Contribute-To-This-Project#next-steps).

**Comment by sappenfield** [comment]
Much thanks Mazen!

**Pull request merged**